nms/packages/magmalte/app/views/subscriber/__tests__/SubscriberTest.js

```diff
@@ -38,6 +38,7 @@ const subscribers = {
   IMSI0000000000: {
     name: 'subscriber0',
     active_apns: ['oai.ipv4'],
+    allowed_nw_types: ['NT_EPC', 'NT_5GC'],
```

Comment:
Replace these with reference to `NETWORK_TYPE` // `NetworkType`

Reply:
I'm referencing it like this.
`const coreNetworkTypes = Object.keys(CoreNetworkTypes).map(key => CoreNetworkTypes[key]);`
`allowed_nw_types: coreNetworkTypes`